feat(engine): 2393: : explain "Duplicate function implementation" error (#79)
mattpocock/ts-error-translator: packages/engine/errors/2393.md

---
original: "Duplicate function implementation."

> excerpt: "You've already declared a function having similar name."

Reviewer: ```suggestion
excerpt: "You've already declared a function with the same name."
```
